### PR TITLE
{tools}[foss/2016b] behave 1.2.5 w/ Python 2.7.12

### PR DESCRIPTION
**Pull request by akesandgren**
This is a nice little piece of software.
We use it for our vasp test suite, developed by Peter Larsson, NSC LiU

Description of behave:

Behavior-driven development (or BDD) is an agile software development
technique that encourages collaboration between developers, QA and
non-technical or business participants in a software project.

### CODE REVIEW

File: easybuild/easyconfigs/b/behave/behave-1.2.5-foss-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = "PythonPackage"
+
+name = 'behave'
+version = '1.2.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pytest.org'
+homepage = 'https://pypi.python.org/packages'
```
Comment (boegel) [inline easybuild/easyconfigs/b/behave/behave-1.2.5-foss-2016b-Python-2.7.12.eb]: @akesandgren what's the right homepage?

File: easybuild/easyconfigs/b/behave/behave-1.2.5-foss-2016b-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+software project."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+#    ('py', '1.4.31', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/b/behave/behave-1.2.5-foss-2016b-Python-2.7.12.eb]: in or out?

File: easybuild/easyconfigs/b/behave/behave-1.2.5-foss-2016b-Python-2.7.12.eb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+moduleclass = 'tools'
+
```
Comment (boegel) [inline easybuild/easyconfigs/b/behave/behave-1.2.5-foss-2016b-Python-2.7.12.eb]: drop empty line please :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 15 out of 15 (1 easyconfigs in this PR)
node2081.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c91238c18ed5e2ce400e55ab13b74cd8 for a full test report.

**Comment by akesandgren** [comment]
Changes made as per comments.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2301.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/afcf05f63357adfc9fc97d8a2cd0a433 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**